.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+Pull request overview
+---------------------
+Please change this line to a description of the pull request, with useful supporting information including whether it is a new feature, or fixes a defect, a cross reference to any defects addressed in this PR, the type of changes to be made, and whether diffs are expected/justified based on this change.
+
+### Work Checklist
+Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
+ - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
+ - [ ] Appropriate labels added to this PR to be consumed into the changelog (currently Defect, Refactoring, NewFeature, Performance, and DoNotPublish are supported)
```

Comment:
Do we want to include an explanation of the labels? DoNotPublish has caused some confusion in the past, maybe here is the place to quash that.